<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Thresh type</string>
    </property>
    <widget class="QRadioButton" name="radioButton_binary">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BINARY</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_binary_inv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BINARY_INV</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_trunc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TRUNC</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tozero">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TOZERO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tozero_inv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TOZERO_INV</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_thresh_types">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All Thresh Types</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>22</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_threshold">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshhold:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Adaptive Method</string>
    </property>
    <widget class="QRadioButton" name="radioButton_none">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_otsu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OTSU</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_adap_mean">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ADAPTIVE_MEAN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_adap_gauss">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ADAPTIVE_GAUSS</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_adap_methods">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>All Adap Methods</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
